examples/mixPureFusionMaterials.py

```diff
@@ -2,6 +2,7 @@
 import os
 
 import material_db_tools as mdbt
+from pyne.material_library import MaterialLibrary
```

Comment:
Curious... since `material_db_tools` imports this already, can we access it as `mdbt.MaterialLibrary`??

Reply:
I just tested this and I think it works - not sure what the best practice is...